src/features/monthly-budget/BudgetTable.js

```diff
@@ -37,7 +38,18 @@ export default function BudgetTable({
             <tr title={budget.tooltip ? budget.tooltip : null}>
               <td>{budget.name}</td>
               <td>
-                <span className="text-muted">R$</span> {budget.amount}
+                <NumberFormat
+                  value={Number(budget.amount)}
+                  defaultValue={Number(budget.amount)}
+                  displayType="text"
+                  fixedDecimalScale
+                  decimalSeparator={','}
+                  thousandSeparator={'.'}
+                  decimalScale={2}
+                  renderText={value => (
```

Comment:
It's just a suggestion. What you think about creating a function a use it here? It's just for avoid the code repetition. Something like this:

```js
const renderText = value => (<><span className="text-muted">R$</span> {value}</>)
```

Reply:
Okay, I"ll encapsulate this. but, the other comments need a different data type. I will create a HOC that receives the dependency data